aws/build.gradle

```diff
@@ -112,6 +112,8 @@ task deploy(type: AWSLambdaInvokeTask) {
   "max_cores": ${ext.getParameterValue('maxCores') ?: 0},
   "min_memory": ${ext.getParameterValue('minMemory') ?: 0},
   "max_memory": ${ext.getParameterValue('maxMemory') ?: 0},
+  "min_cpu_usage": ${ext.getParameterValue('minCpuUsage') ?: 1},
+  "max_inactive_time_interval": ${ext.getParameterValue('maxInactiveTimeInterval') ?: 5},
```

Comment:
I would change the names of parameters to include 'stuckGuard', i.e. stuckGuardMinCpuUsage and stuckGuardMaxInactiveTimeInterval to reduce the possible confusion.

Reply:
Hi @nikolayilyin , I have changed the parameter names, please take a look again.

For the documentation, I couldn't find the correct place, do you mean here?
https://beam.readthedocs.io/en/latest/developers.html#automated-cloud-deployment with the parameter part?

Reply:
Yes, the link you shared contains the description of 'deploy' command, let's add the description of new parameters to that comand

Reply:
Hi @nikolayilyin , I did a commit to add description of the deploy command.
And also I did a rebase from develop branch, but now the changes of PR looks mess, only three files is changed by me
![image](https://user-images.githubusercontent.com/11383105/194872583-2e1d8b66-a083-460b-ae4c-81ef6191abf1.png)

Do you have any suggestions of the default values? stuckGuardMinCpuUsage:1, stuckGuardMaxInactiveTimeInterval:5

Reply:
The default values looks OK for me, let's keep them as 5 and 1 for now

Reply:
Can we close the issue and merge the PR? Or what is the process for BEAM project?